src/manage.c

```diff
@@ -7758,3 +7908,115 @@ delete_resource (const char *type, const char *resource_id, int ultimate)
   assert (0);
   return -1;
 }
+
+
+/* Inter-Process Communication */
```

Comment:
Could we move this section to a new file? It looks self contained and manage.c is already large.